SUPPORT.md

```diff
@@ -0,0 +1,9 @@
+Need help or support?
+------------------
+
+ **Please note:**
+- Only submit issues for bug reports, feature requests or enhancements.
+- Grafana project uses GitHub mainly for tracking bugs and feature requests.
+- Asking a question by opening an issue will directly result in issue being closed.
+
+If you require help or support then ask a question and/or find existing questions/answers in the [Grafana community site](https://community.grafana.com/).
```

Comment:
Should we mention paid support/enterprise here somehow or would that be too much?

Reply:
I'm thinking no for now, but we could evaluate to add that later.